content/guides/shimming.md

```diff
@@ -158,3 +159,86 @@ module.exports = {
 ## Node Built-Ins
 
 Node built-ins, like `process`, can be polyfilled right directly from your configuration file without the use of any special loaders or plugins. See the [node configuration page](/configuration/node) for more information and examples.
+
+
+## Loading polyfills on demand
+
+It's really common in web projects to include polyfills in the main bundle. This is not recommended since we are penalizing modern browsers users, since they will have to download a bigger file unnecessarely.
```

Comment:
@jeremenichelli did you see this comment? I see the other changes but this sentence looks the same and "unnecessarely" is still spelled wrong (except that now it's highlighted 😆 ). Aside from this I think this PR is pretty much good to go, though we will have to rebase to resolve conflicts.